src/community/resources-and-tools/index.md.njk

```diff
@@ -34,6 +34,9 @@ SVG images for creating flow diagrams of GOV.UK services.
 
 ## Create prototypes and wireframes
 
+[MURAL flow diagram](https://github.com/link-tbc) -
```

Comment:
The link still isn't in here...

Then a couple of small details, sorry for being pernickety!

1. I'm not sure whether they prefer to stylise it as MURAL or Mural, but either way we should be consistent here
2. We order the resources alphabetically, so it should move to under the Figma link

Happy to fix all this myself if easier for you.